docker/lib/dependabot/shared/shared_file_parser.rb

```diff
@@ -42,7 +48,7 @@ def source_from(parsed_line)
 
         source[:registry] = parsed_line.fetch("registry") if parsed_line.fetch("registry")
         source[:tag] = parsed_line.fetch("tag") if parsed_line.fetch("tag")
-        source[:digest] = parsed_line.fetch("digest") if parsed_line.fetch("digest")
+        source[:digest] = "sha256:#{parsed_line.fetch('digest')}" if parsed_line.fetch("digest")
```

Comment:
IMO adding `sha256` is not generally true. See https://github.com/opencontainers/image-spec/blob/main/descriptor.md#digests

Reply:
Agreed, however as of right now, it appears that the updater is pretty much hardcoded to using `sha256`. I'm down to change it, but that would be a lot more work than what I have here